src/cli/one_helper/onevnet_helper.rb

```diff
@@ -461,6 +498,16 @@ def format_resource(vn, options = {})
             puts format('%-15s', id)
         end
 
+        puts
+
+        CLIHelper.print_header(str_h1 % 'VIRTUAL MACHINES', false)
+
+        updated, outdated, error = vn.vm_ids
+
+        puts format(str, 'UPDATED', updated.join(','))
+        puts format(str, 'OUTDATED', outdated.join(','))
+        puts format(str, 'ERROR', error.join(','))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against an allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html